src/pykeen/metrics/ranking.py

```diff
@@ -1,6 +1,87 @@
 # -*- coding: utf-8 -*-
 
-"""Ranking metrics."""
+"""
+Ranking metrics.
+
+This module comprises various rank-based metrics, which get an array of individual ranks as input, as summarize them
+into a single-figure metric measuring different aspects of ranking performance.
+
+We can generally distinguish:
+
+Base Metrics
+------------
+These metrics directly operate on the ranks:
+
+The following metrics measures summarize the central tendency of ranks
+
+- :class:`pykeen.metrics.ranking.ArithmeticMeanRank`
+- :class:`pykeen.metrics.ranking.GeometricMeanRank`
+- :class:`pykeen.metrics.ranking.HarmonicMeanRank`
+- :class:`pykeen.metrics.ranking.MedianRank`
+
+The Hits at K metric is closely related to information retrieval and measures the fraction of times when the correct
+result is in the top-$k$ ranked entries, i.e., the rank is at most $k$
+
+- :class:`pykeen.metrics.ranking.HitsAtK`
+
+The next metrics summarize the dispersion of ranks
+
+- :class:`pykeen.metrics.ranking.MedianAbsoluteDeviation`
+- :class:`pykeen.metrics.ranking.Variance`
+- :class:`pykeen.metrics.ranking.StandardDeviation`
+
+and finally there is a simple metric to store the number of ranks which where aggregated
+
+- :class:`pykeen.metrics.ranking.Count`
+
+Inverse Metrics
+---------------
+The inverse metrics are reciprocals of the central tendency measures. They offer the advantage of having a fixed value
+range of $(0, 1]$, with a known optimal value of $1$:
+
+- :class:`pykeen.metrics.ranking.InverseArithmeticMeanRank`
+- :class:`pykeen.metrics.ranking.InverseGeometricMeanRank`
+- :class:`pykeen.metrics.ranking.InverseHarmonicMeanRank`
+- :class:`pykeen.metrics.ranking.InverseMedianRank`
+
+Adjusted Metrics
+----------------
+Adjusted metrics build upon base metrics, but adjust them for chance, cf. [berrendorf2020, hoyt2022]_. All adjusted
```

Comment:
@cthoyt where would you put these references?

Reply:
in the references.rst

Reply:
Done in 72f9c56f